crates/debug/types/src/source_file.rs

```diff
@@ -625,18 +625,27 @@ impl SourceContent {
         line_index: LineIndex,
         column_index: ColumnIndex,
     ) -> Option<ByteIndex> {
-        let column_index = column_index.to_usize();
         let line_span = self.line_range(line_index)?;
         let line_src = self
             .content
             .get(line_span.start.to_usize()..line_span.end.to_usize())
             .expect("invalid line boundaries: invalid utf-8");
-        if line_src.len() < column_index {
+
+        let col_chars = column_index.to_usize();
```

Comment:
*Nit:* This name is less clear than `column_index` (the previous name of this binding) - the `ColumnIndex` type is a 0-indexed value representing the index of the character on the line, but `col_chars` could be interpreted a number of different ways. I would keep the old name.